message tab_CharacterAttrTable 
{
	optional string tname = 1;
	repeated tab_CharacterAttr tlist = 2;
}

message tab_CharacterAttr
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 Level = 3;
	optional int32 Type = 4;
	optional int32 InGameName = 5;
	optional int32 TrainCost = 6;
	optional int32 UpgradeCost = 7;
	optional int32 UpgradeTime = 8;
	optional int32 UpdateDiamond = 9;
	optional int32 Hurt = 10;
	optional int32 Velocity = 11;
	optional int32 Hp = 12;
	optional int32 CostUnit = 13;
	optional int32 FightCapacity = 14;
}